alert-database/src/test/java/com/synopsys/integration/alert/database/api/DefaultCustomCertificateAccessorTest.java

```diff
@@ -50,7 +53,7 @@ public void getCertificateTest() {
 
     @Test
     public void storeCertificateTest() throws Exception {
-        CustomCertificateModel certificateModel = new CustomCertificateModel(alias, content, new Date().toString());
+        CustomCertificateModel certificateModel = new CustomCertificateModel(alias, content, testDate.toString());
```

Comment:
Why are we using the DateUtils above and the .toString() method here? We should be more consistent and choose one of these approaches.